webpack.config.js

```diff
@@ -63,6 +63,18 @@ const config = {
 };
 
 module.exports = (env) => {
+
+  if (env.onPrem) {
+    console.log("building for onprem");
+    config.entry.extension = "./src/extensionOnPrem.ts";
+    config.plugins.push(new webpack.DefinePlugin({
+      __ONPREM__: JSON.stringify(true)
```

Comment:
why here and not in the action ?

Reply:
GitHub action? How could it be in the action, when the web pack is doing the replacements